src/importlinter/contracts/layers.py

```diff
@@ -16,27 +16,30 @@
 
 
 @dataclass(frozen=True)
-class Layer:
+class ModuleTail:
     name: str
     is_optional: bool = False
 
 
+@dataclass(frozen=True)
+class Layer:
```

Comment:
This is a potentially breaking change, if a developer has built a custom contract type which uses either `LayerField` or the `Layer` class. That being said, I do not know if any such developer actually exists!

I think it's a good change though. I'm inclined to leave as-is, but include a note in the CHANGELOG and release this with a major version bump.

Suggested text for CHANGELOG:

```
- In ``importlinter.contracts.layers``, ``Layer`` and ``LayerField`` 
  have changed their API slightly. This could impact custom
  contract types depending on these classes. 
```

Reply:
Yes, added this line to the changelog